wordpress/wp-content/plugins/cds-base/languages/cds-snc-fr_CA.po

```diff
@@ -280,14 +277,14 @@ msgstr "Envoyez une copie à votre courriel."
 
 #: classes/Modules/Forms/Contact/ContactForm.php:196
 msgid "Submit"
-msgstr "Soumettez"
+msgstr "Soumettre"
 
 #: classes/Modules/Forms/Contact/Setup.php:50
 #: classes/Modules/Forms/RequestSite/RequestSiteForm.php:25
 #: classes/Modules/Forms/RequestSite/Setup.php:64
-#: classes/Modules/Forms/Subscribe/Setup.php:122
+#: classes/Modules/Forms/Subscribe/Subscribe.php:100
 msgid "Please complete the required field(s) to continue"
-msgstr "Veuillez remplir le(s) champ(s) obligatoire pour continuer"
+msgstr "Veuillez remplir le champ obligatoire pour continuer"
 
```

Comment:
The french translation here loses the nuance that you can select more than one, but it's a pretty minor thing overall and hopefully people understand checkboxes.

Reply:
🤷